yt_dlp/extractor/bilibili.py

```diff
@@ -774,3 +775,119 @@ def _real_extract(self, url):
         return self.url_result(
             'http://www.bilibili.tv/video/av%s/' % video_id,
             ie=BiliBiliIE.ie_key(), video_id=video_id)
+
+
+class BiliIntlIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?bili(?P<type>bili\.tv|intl.com)/(?:[a-z]{2}/)?play/(?P<season_id>\d+)/(?P<id>\d+)'
+    _TESTS = [{
+        'url': 'https://www.bilibili.tv/en/play/34613/341736',
+        'info_dict': {
+            'id': '341736',
+            'ext': 'mp4',
+            'title': 'The First Night',
+            'thumbnail': 'https://i0.hdslb.com/bfs/intl/management/91e30e5521235d9b163339a26a0b030ebda54310.png',
+            'episode_number': 2,
+        },
+        'params': {
+            'format': 'bv',
+        },
+    }, {
+        'url': 'https://www.biliintl.com/en/play/34613/341736',
+        'info_dict': {
+            'id': '341736',
+            'ext': 'mp4',
+            'title': 'The First Night',
+            'thumbnail': 'https://i0.hdslb.com/bfs/intl/management/91e30e5521235d9b163339a26a0b030ebda54310.png',
+            'episode_number': 2,
+        },
+        'params': {
+            'format': 'bv',
+        },
+    }]
+
+    def _real_extract(self, url):
+        type, season_id, id = re.match(self._VALID_URL, url).groups()
```

Comment:
```suggestion
        type, season_id, id = self._match_valid_url(url).groups()
```

Reply:
afaict this is only for the py2 compat. It does the same thing.
Since py2 is no longer supported and re is already imported, i think it's okay to use this

Reply:
For anyone else reading this:
`self._match_valid_url(url)` is effectively the same as `re.match(self._VALID_URL, url)` and exist only for standardizing this operation across extractors (similar to self._match_id)